### PR TITLE
[xml] russian.xml  v 8.7.1

### PR DESCRIPTION
**Pull request by GOLEM777**
@donho
Patch queue conflict.
 This has happened before.
Problem due cd45afc
The text has been changed out of order of correction.

### CODE REVIEW

File: PowerEditor/installer/nativeLang/russian.xml [hunk 1]
```diff
@@ -550,13 +550,13 @@ Updated to v8.7.0:
 				<Item id="1688" name="Всегда"/>
 				<Item id="1703" name="&amp;. - новая строка"/>
 				<Item id="1722" name="О&amp;братное направление поиска"/>
-				<Item id="1721" name="▲"/>
-				<Item id="1723" name="▼"/>
+				<Item id="1721" name="^"/>
+				<Item id="1723" name="Ў"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/russian.xml:554]: Is it intentional change?

File: PowerEditor/installer/nativeLang/russian.xml [hunk 2]
```diff
@@ -550,13 +550,13 @@ Updated to v8.7.0:
...
 				<Item id="1725" name="&amp;Коп. Помеченный Текст"/>
 				<Menu>
-					<Item id="1726" name="⇅ Поменять местами Найти и Заменить"/>
-					<Item id="1727" name="⤵ Скопировать из Найти в Заменить"/>
-					<Item id="1728" name="⤴ Скопировать из Заменить в Найти"/>
+					<Item id="1726" name="? Поменять местами Найти и Заменить"/>
+					<Item id="1727" name="? Скопировать из Найти в Заменить"/>
+					<Item id="1728" name="? Скопировать из Заменить в Найти"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/russian.xml:559]: Is it intentional change?

File: PowerEditor/installer/nativeLang/russian.xml [hunk 3]
```diff
@@ -1020,7 +1021,7 @@ Updated to v8.7.0:
 					<Item id="6123" name="Язык интерфейса"/>
 					<Item id="6128" name="Альтернативные значки"/>
 					<Item id="6131" name="Панель Меню"/>
-					<Item id="6132" name="Скрыть кнопки ＋ ▼ ✕ в строке меню (Необходим перезапуск Notepad++)"/>
+					<Item id="6132" name="Скрыть кнопки + Ў ? в строке меню (Необходим перезапуск Notepad++)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/russian.xml:1024]: Is it intentional change?

**Review by donho — CHANGES_REQUESTED** [review]
There are some intentional changes to me.

**Comment by GOLEM777** [comment]
Sorry! Yes, I was in a hurry. ))

**Review by donho — APPROVED** [review]

**Pull request closed without merging**